lib/reversetunnel/track/tracker.go

```diff
@@ -0,0 +1,303 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package track
+
+import (
+	"context"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/gravitational/teleport/lib/utils"
+	"github.com/gravitational/teleport/lib/utils/workpool"
+)
+
+type Lease = workpool.Lease
```

Comment:
The `Lease` type is the only part  of the `workpool` API which isn't wrapped.  Alias unifies the API and eliminates the need for consumer to import the `workpool` package, which is otherwise an implementation detail.